DPGAnalysis/SiStripTools/test/OccupancyPlotsTest_phase2_cfg.py

```diff
@@ -245,20 +259,20 @@
 
 #----GlobalTag ------------------------
 
-#process.load("Configuration.StandardSequences.GeometryDB_cff")
 #process.load('Configuration.Geometry.GeometryExtendedPhase2TkBE5DPixel10DReco_cff')
-process.load('Configuration.Geometry.GeometryExtended2023MuonReco_cff')
+#process.load('Configuration.Geometry.GeometryExtended2023MuonReco_cff')
+process.load('Configuration.Geometry.GeometryExtended2023MuondevReco_cff')
 process.load('Configuration.StandardSequences.MagneticField_38T_PostLS1_cff')
-#process.load("Configuration.Geometry.GeometryExtendedPhaseIPixelReco_cff")
-#process.load("Configuration.Geometry.GeometryExtendedPhaseIPixel_cff")
 process.load("Configuration.StandardSequences.Reconstruction_cff")
 
 from SLHCUpgradeSimulations.Configuration.phase2TkCustomsBE5DPixel10D import *
 
 process.load("Configuration.StandardSequences.FrontierConditions_GlobalTag_cff")
+if not options.globalTag :
+  #from Configuration.AlCa.GlobalTag import GlobalTag
+  #process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:upgradePLS3', '')
+  options.globalTag = "DES23_62_V1::All"
```

Comment:
You can set the default values of the "options" parameters, when the options parameters are defined at the beginning of the cfg file